scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -110,6 +110,48 @@ namespace <%= namespace %> {
             });
         }
 
+        private void CheckoutWithTokenizedPayment(TokenizedPaymentInput tokenizedPaymentInput, Action<ShopifyError> callback) {
```

Comment:
Something I didn't consider in defining the `INativeCheckout` and `IWebCheckout` interfaces is code that is related to a native checkout that is platform-agnostic. For example, this seems like something that:

1. Only is used for Native Payments
2. Can run for both Android/Apply Pay

If this is a pattern we start to see emerging we can define an abstract base class for native pay (`BaseNativeCheckout`) that implements this code. This way we can keep the checkout logic out of the Cart and delegate to the checkout objects.